pkg/framework/plugins/removepodsviolatingtopologyspreadconstraint/topologyspreadconstraint.go

```diff
@@ -47,30 +48,44 @@ type topology struct {
 	pods []*v1.Pod
 }
 
-// nolint: gocyclo
-func RemovePodsViolatingTopologySpreadConstraint(
-	ctx context.Context,
-	client clientset.Interface,
-	strategy api.DeschedulerStrategy,
-	nodes []*v1.Node,
-	podEvictor *evictions.PodEvictor,
-	evictorFilter *evictions.EvictorFilter,
-	getPodsAssignedToNode podutil.GetPodsAssignedToNodeFunc,
-) {
-	strategyParams, err := validation.ValidateAndParseStrategyParams(ctx, client, strategy.Params)
+// RemovePodsViolatingTopologySpreadConstraint evicts pods which violate their topology spread constraints
+type RemovePodsViolatingTopologySpreadConstraint struct {
+	handle    framework.Handle
+	args      *componentconfig.RemovePodsViolatingTopologySpreadConstraintArgs
+	podFilter podutil.FilterFunc
+}
+
+var _ framework.BalancePlugin = &RemovePodsViolatingTopologySpreadConstraint{}
```

Comment:
Missing `var _ framework.Plugin = &RemovePodsViolatingTopologySpreadConstraint{}` check

Reply:
doesn't BalancePlugin implement it? is it necessary?

Reply:
You are right :) Please ignore my comment.